fendermint/fendermint/vm/interpreter/src/fvm/state/query.rs

```diff
@@ -204,6 +205,25 @@ where
         &self.state_params
     }
 
+    /// Returns the registry of built-in actors as enrolled in the System actor.
+    pub async fn builtin_actors(self) -> anyhow::Result<(Self, Vec<(String, Cid)>)> {
```

Comment:
Eventually we may want to add a `debug_ipcBuiltinActors` JSON-RPC operation to return known built-in actors for debugging.